### PR TITLE
Nollning fonts

### PR DESCRIPTION
**Pull request by navro**

### CODE REVIEW

File: app/views/admin/pages/_form.html.erb [hunk 1]
```diff
@@ -2,8 +2,7 @@
   <%= f.input :title_sv, wrapper_html: {class: 'inline half'} %>
   <%= f.input :title_en, wrapper_html: {class: 'inline half'} %>
   <%= f.input :url %>
-  <%= f.input :namespace, collection: Page.namespaces, input_html: { class: 'select2_tags' },
-              include_blank: false %>
+  <%= f.input :namespace, collection: Page.namespaces, input_html: { class: 'select2_tags' } %>
```
Comment (davidwessman) [inline app/views/admin/pages/_form.html.erb]: Requiring a namespace? ;)

Comment (navro) [inline app/views/admin/pages/_form.html.erb]: > Requiring a namespace? ;)

No. But blank should be allowed. It is not now.

Comment (davidwessman) [inline app/views/admin/pages/_form.html.erb]: Oh, nice 😄

**Comment by davidwessman** [comment]
:+1:

**Pull request merged**